static/app/utils/analytics/tracingEventMap.tsx

```diff
@@ -53,6 +65,10 @@ export type TracingEventParameters = {
     visualizes: BaseVisualize[];
     visualizes_count: number;
     attribute_breakdowns_mode?: 'breakdowns' | 'cohort_comparison';
+    cross_event_log_queries?: string[];
+    cross_event_metric_queries?: string[];
+    cross_event_query_count?: number;
+    cross_event_span_queries?: string[];
```

Comment:
Given the type is not very restrictive, are there cardinality concerns here with a `string[]`? Looking at the full type of `'trace.explorer.metadata'`, it seems like we are already putting a bunch of string values in, but I wanted to make sure!